doc/design/syntax.md

```diff
@@ -115,10 +115,10 @@ The following words are keywords which are used as syntactic elements, and canno
 purposes.
 
 ```hemlock
-and      external  match   val
-also     false     module  when
-as       for       of      while
-assert   fun       open    with
+and      external  match       val
+also     false     mutability  when
+as       for       of          while
+assert   fun       open        with
```

Comment:
This is actually a really annoying table, because Markdown doesn't provide any sort of tabular output that keeps the keyword list from being ridiculous. So it's manually formatted.